core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -105,7 +105,7 @@ protected function doReplace($strBuffer, $blnCache)
 		for ($_rit=0, $_cnt=\count($tags); $_rit<$_cnt; $_rit+=2)
 		{
 			$strBuffer .= $tags[$_rit];
-			$strTag = $tags[$_rit+1];
+			$strTag = $tags[$_rit+1] ?? '';
```

Comment:
How is this related to the PR?

Reply:
`$tags[$_rit+1]`  can be an invalid array access but the old code doesn't care. This is only needed for the integration tests that would break otherwise because they are running in strict mode. :neutral_face: